$OpenBSD: patch-UI_Scheduler_UIxCalListingActions_m,v 1.3 2017/05/09 20:50:06 sebastia Exp $

Do not try to access a method on nil object, it horribly crashes/breaks
see PR: https://github.com/inverse-inc/sogo/pull/224

--- UI/Scheduler/UIxCalListingActions.m.orig	Fri Mar 10 17:23:11 2017
+++ UI/Scheduler/UIxCalListingActions.m	Tue May  9 12:34:39 2017
@@ -493,7 +493,7 @@ static NSArray *tasksFields = nil;
               for (i = 0; i < count; i++)
                 {
                   currentInfo = [newInfoForComponent objectAtIndex: i];
-                  if (![currentInfo isEqual:[NSNull null]] && [currentInfo respondsToSelector: @selector (stringByEscapingHTMLString)])
+                  if ([currentInfo respondsToSelector: @selector (stringByEscapingHTMLString)])
                     [newInfoForComponent replaceObjectAtIndex: i withObject: [currentInfo stringByEscapingHTMLString]];
                 }
               [infos addObject: newInfoForComponent];
